### DISCUSSION TITLE
M365 계정 관련

### DISCUSSION THREAD

**Discussion by zer0big** [fusiondevkr/pl900#28] [1 Q&A]
안녕하세요?

저의 경우 이미 E5 계정(Microsoft 365 developer subscriptions)을 가지고 있는데 이를 활용해도 될까요?

**Comment by justinyoo** [comment]
@zer0big 활용하셔도 상관은 없습니다만, 향후 실습을 할 때 설정이 달라서 생기는 문제에 대해 대응을 하기가 어려울 수도 있습니다.

**Reply by zer0big** [reply]
아, 그렇겠네요~ 알겠습니다.

**Reply by sesme100** [reply]
사용자께서 기존 조직의 E5 계정을 사용하여 활용하는 경우에는 조직의 보안과 규정에 의하여 연결이 되지 않는 데이터들이 있을 수 도 있습니다. (주로 조직의 Admin에 의하여 제한되어 있습니다) 
M365 개발자 계정과 파워플랫폼 계정의 경우 Data Connector에 관련된 보안과 Policy 규정을 본인이 Admin으로 직접 설정하시게 됩니다. 그러므로 개발에 있어서 파워 플랫폼을 훨씬 자유롭게 사용하실 수 있습니다.
